Source/WebCore/dom/CustomElementReactionQueue.h

```diff
@@ -233,7 +233,7 @@ class CustomElementReactionStack : public CustomElementReactionDisallowedScope::
 private:
     WEBCORE_EXPORT void processQueue(JSC::JSGlobalObject*);
 
-    CustomElementQueue* m_queue { nullptr }; // Use raw pointer to avoid generating delete in the destructor.
+    std::unique_ptr<CustomElementQueue> m_queue;
```

Comment:
Did we perf test this??

Reply:
No, I'll do so now.

Reply:
Personally I had trouble understanding why it would be slower. Surely we were destroying the queue before or we'd have a leak. How is using std::unique_ptr any lower than new/delete?

Reply:
Oh, the issue was that every ~CustomElementReactionStack has to generate a call to ~CustomElementQueue in addition to calling processQueue (i.e. will have two branches instead of one).